.travis.yml

```diff
@@ -2,24 +2,26 @@ language: generic
 services:
   - docker
 
-notifications:
-  email:
-    on_success: always
-    on_failure: always
+cache:
+  directories:
+    - $HOME/.ccache
+
+git:
+  quiet: true
+
 env:
+  global:
+    - CCACHE_DIR=$HOME/.ccache
   matrix:
-    - ROS_DISTRO="kinetic"   ROS_REPO=ros
-    - ROS_DISTRO="kinetic"   ROS_REPO=ros-shadow-fixed
-    - ROS_DISTRO="kinetic"   PRERELEASE=true
-    - ROS_DISTRO="melodic"   ROS_REPO=ros
-    - ROS_DISTRO="melodic"   ROS_REPO=ros-shadow-fixed
-    - ROS_DISTRO="melodic"   PRERELEASE=true
-matrix:
-  allow_failures:
-    - env: ROS_DISTRO="melodic"   ROS_REPO=ros
-    - env: ROS_DISTRO="melodic"   ROS_REPO=ros-shadow-fixed
-    - env: ROS_DISTRO="melodic"   PRERELEASE=true
+    - ROS_DISTRO="kinetic" ROS_REPO=main
+    - ROS_DISTRO="kinetic" ROS_REPO=ros-testing
```

Comment:
`ROS_REPO=testing` in the new naming convention..

`ros-testing` is the actually name of the ROS1 repository on the server..
For now it is treated like `testing`, but this might change in the future.

Reply:
I changed that based on a notice from `industrial_ci` in the first job that was run I believe.

Anyway, I'll change it.

Reply:
https://github.com/ros-industrial/industrial_ci/blame/master/industrial_ci/src/env.sh#L91 ?

Reply:
Probably, I cannot find it in the build log any more.

I must have misremembered.